frontend/Auth/FinishResetPassword.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import { Button, ButtonKind } from "../Styleguide/Button";
+import { FormField } from "../Styleguide/FormField";
+import { FormFieldLabel } from "../Styleguide/FormFieldLabel";
+import { Input } from "../Styleguide/Input";
+import { RouterProps } from "react-router";
+import { useMutation, useQuery } from "react-query";
+import { flaxFetch } from "../Utils/flaxFetch";
+import { unary } from "lodash-es";
+
+export function FinishResetPassword(props: RouterProps) {
+  const [finishResetPasswordFormData, setFinishResetPasswordFormData] =
+    useState<FinishResetPasswordFormData>({
+      password: "",
+    });
+
+  const paramString = props.history.location.search;
+  let searchParams = new URLSearchParams(paramString);
+  const token = searchParams.get("jwt") || "MISSING TOKEN";
+
+  const queryFunctionhelper = (queryKey) => {
+    const token = queryKey["queryKey"][0];
+    return flaxFetch<object>(
+      `/validate-token/${token}?tokenType=passwordReset`,
+      {
+        method: "GET",
+      }
+    );
+  };
+
+  const info = useQuery(token, queryFunctionhelper);
+
+  const submitMutation = useMutation<
+    void,
+    Error,
+    React.FormEvent<HTMLFormElement>
+  >((evt) => {
+    evt.preventDefault();
+    let requestPromise = flaxFetch<void>(`/send-reset-password-email`, {
+      method: "POST",
+      body: {
+        password: finishResetPasswordFormData.password,
+        token: token,
+      },
+    });
+    return requestPromise;
+  });
+
+  return (
+    <div className="flex justify-center h-screen p-24 sm:pt-80">
+      <form
+        onSubmit={unary(submitMutation.mutate)}
+        className="relative lg:max-w-sm"
+      >
+        <h1 className="text-gray-500 place-self-start mb-2 text-5xl lg:text-xl">
+          Set New Password
+        </h1>
+        <p className="text-left text-gray-600 py-8 text-4xl lg:text-sm">
+          Enter your new password below.
+        </p>
+        <FormField className="mb-40">
+          <FormFieldLabel className="text-3xl lg:text-xs" htmlFor="password">
+            Password
+          </FormFieldLabel>
+          <Input
+            id="password"
+            className="text-5xl lg:text-sm"
+            type="password"
+            value={finishResetPasswordFormData.password}
+            onChange={(evt) =>
+              setFinishResetPasswordFormData({
+                ...finishResetPasswordFormData,
+                password: evt.target.value,
+              })
+            }
+            required
+          />
+        </FormField>
+
+        <div className="absolute inset-x-0 my-8 bottom-0">
```

Comment:
It's a little surprising to me that absolute positioning was needed for the Submit New Password button. Why was this needed?

Reply:
It was needed to ensure that the button stayed at the bottom.  Without it rises to just below the input.
<img width="298" alt="with-absolute" src="https://user-images.githubusercontent.com/61174160/146476208-b0b3c583-ffa6-4fcb-b8f3-f2ac7b835435.png">
<img width="246" alt="without-absolute" src="https://user-images.githubusercontent.com/61174160/146476211-5b06a6d3-8245-413a-b32f-b921e093c229.png">

Reply:
I probably would accomplish this with `display: flex` + `flex-direction: column` + `justify-content: space-between`. Using absolute position is fine but it generally complicates things when adding new features or maintaining things. Specifically, having to account for the button when creating padding/margin can be a pain, scroll behaviors are a lot more difficult, and also creating absolute positioned elements relative to the whole body/document is more difficult. So please switch to using the css i described above

Reply:
Removed absolute.  Changed css.  Works great.  Thank you!